### PR TITLE
[GNA] Added tests for cases connection split->concat

### PR DESCRIPTION
**Pull request by admitriev-gna**
@p-rozen Can you review these tests?

### CODE REVIEW

File: inference-engine/tests/functional/plugin/gna/pass_tests/insert_copy_layer_split_concat.cpp [hunk 1]
```diff
@@ -0,0 +1,96 @@
+// Copyright (C) 2020 Intel Corporation
```
Comment (dorloff) [inline inference-engine/tests/functional/plugin/gna/pass_tests/insert_copy_layer_split_concat.cpp:1]: 2021

File: inference-engine/tests/functional/plugin/shared/include/subgraph_tests/multiple_connect_split_concat.hpp [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2020 Intel Corporation
```
Comment (p-rozen) [inline inference-engine/tests/functional/plugin/shared/include/subgraph_tests/multiple_connect_split_concat.hpp:1]: 2021

**Review by p-rozen — APPROVED** [review]
just change the copyright dates

File: inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_connect_split_concat.cpp [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+void MultipleConnectSplitConcatTest::SetUp() {
+    InferenceEngine::Precision netPrecision;
+    std::tie(netPrecision, targetDevice, configuration) = this->GetParam();
+    auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+
+    auto params = ngraph::builder::makeParams(ngPrc, {{1, 256}});
+    auto relu_start = std::make_shared<ngraph::opset1::Relu>(params[0]);
+    auto split = ngraph::builder::makeSplit(relu_start, ngPrc, 2, 1);
+    auto const_concat = ngraph::builder::makeConstant(ngPrc, {1, 96}, std::vector<float>{0});
+    auto const_concat_2 = ngraph::builder::makeConstant(ngPrc, {1, 96}, std::vector<float>{0});
```
Comment (p-rozen) [inline inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_connect_split_concat.cpp:32]: After looking deeper the setup should be like this:
split with 1 output
concat with 2 inputs that are using the same split output

Comment (admitriev-gna) [reply inference-engine/tests/functional/shared_test_classes/src/subgraph/multiple_connect_split_concat.cpp]: fixed as recommended

**Pull request merged**